templates/ubuntu1804.yaml

```diff
@@ -34,14 +34,6 @@ metadata:
     flavor.template.cnv.io/small: "true"
 objects:
 - apiVersion: kubevirt.io/v1alpha2
-  kind: VirtualMachineInstancePreset
-  metadata:
-    name: ubuntu1804
-  spec:
-    selector:
-      matchLabels:
-        kubevirt.io/os: ubuntu1804
-- apiVersion: kubevirt.io/v1alpha2alpha2
```

Comment:
Would be nice to have in the commit message a nice explanation on why this is breaking the ubuntu template.
Is this material to be squashed to one of the previous commits?